pkg/operator/ceph/object/objectstore.go

```diff
@@ -301,3 +314,56 @@ func poolName(storeName, poolName string) string {
 	// the name of the pool is <instance>.<name>, except for the pool ".rgw.root" that spans object stores
 	return fmt.Sprintf("%s.%s", storeName, poolName)
 }
+
+func enableRGWDashboard(context *Context) error {
+	//currently adminuser only for accessing dashboard, in future this will be seperate api
+	dispname := fmt.Sprintf("%s", AdminUser)
+	user := ObjectUser{
+		UserID:      AdminUser,
+		DisplayName: &dispname,
+	}
+	u, errCode, err := CreateUser(context, user, true)
+	if err != nil || errCode != 0 {
+		return errors.Wrapf(err, "error creating admin user %v", err)
+	}
+
+	args := []string{"dashboard", "set-rgw-api-access-key", *u.AccessKey}
+	cephCmd := ceph.NewCephCommand(context.Context, context.ClusterName, args)
+	out, errn := cephCmd.Run()
+	if errn != nil {
+		return errors.Wrapf(errn, "error setting user accesskey %v output: %s", errn, string(out))
```

Comment:
`Wrapf` is already wrapping `errn` (also, why is this `errn` and not the usual `err`?), there is no need to include it in the added info string here.